src/modules/Apiconnectivity.js

```diff
@@ -0,0 +1,69 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const list = document.querySelector('.list-score');
+
+async function startGame(gameName) {
+  const responseStart = await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      name: gameName,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+
+  const responseData = await responseStart.json();
+  return responseData;
+}
+
+async function getScores(gameId) {
+  try {
+    const responseStart = await fetch(`${url}${gameId}/scores/`);
+    const responseData = await responseStart.json();
+    return responseData;
+  } catch (error) {
+    return [];
+  }
+}
+
+async function postScore(gameId, name, score) {
+  if (name === '' || score === '') {
+    alert('Please enter a valid name or Score');
```

Comment:
> Use `arrow functions` instead of the `function keyword`.

- You have done great so far :+1: :+1: But there are some requirements you still need to implement before moving to the next milestone :+1: :100: 
- [ ] According to the project requirement, you are meant to use the arrow functions here instead of the function keyword to define your function, I'll suggest you re-write these functions to use the ES6 arrow function syntax. EG: `const postScore = async (gameId, name, score) => { }`. :+1: :100: